pkg/stub/handler.go

```diff
@@ -34,49 +33,49 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 	return nil
 }
 
-func Reconcile(cluster *logging.ClusterLogging) (err error) {
+func Reconcile(spec *logging.ClusterLogging) (err error) {
 	exists := true
-
+	cluster := k8shandler.NewClusterLogging(spec)
 	// Reconcile certs
-	if exists, cluster = utils.DoesClusterLoggingExist(cluster); exists {
+	if exists = cluster.Exists(); exists {
```

Comment:
we can just shorten this to `if cluster.Exists() {`

Reply:
That previous function also served to get us the most recent iteration of the cluster object so that we could update it without conflicts... does this still cover that case?

Reply:
`cluster` is  a pointer meaning the data associated with the struct to which it points is mutable.  I followed the `sdk.Get` function and the way I interpreted it was that it would 'fill' the struct identified by the pointer. If read correctly then yes its covered.  I would prefer an additional set of eyes to ensure it behaves as I expect.